test/audits/mobile-friendly/viewport.js

```diff
@@ -18,38 +18,22 @@ const assert = require('assert');
 
 /* global describe, it*/
 
-// Need to disable camelcase check for dealing with background_color.
 /* eslint-disable camelcase */
 describe('Mobile-friendly: viewport audit', () => {
```

Comment:
> Presumably we aren't bothered with multiple meta viewport / theme color tags.

Not sure what the browser behavior is in that case. :/  I'd be fine with deferring unless we see the problem hit someone.